lib/jekyll-crds/filters/get_doc_filter.rb

```diff
@@ -16,8 +16,11 @@ def get_docs(objs)
         return nil if objs.nil?
         return get_doc(objs) unless objs.is_a?(Array)
         ids = objs.collect { |obj| obj['id'] }
-        docs = collection_from_obj(objs.first).select { |doc| ids.include?(doc.data['id']) }
-        docs.map(&:to_liquid)
+
+        docs = objs.collect do |obj|
+          collection_from_obj(obj).select {|doc| ids.include?(doc.data['id']) }
+        end
+        docs.flatten.uniq.map(&:to_liquid)
```

Comment:
Totally not necessary, but ruby has a method to flatten and map at the same time. https://apidock.com/ruby/Enumerable/flat_map